lib/utils.js

```diff
@@ -124,15 +168,18 @@ class Utils {
         }
         return os.arch().includes('64') ? 'linux-amd64' : 'linux-386';
     }
-    static getCliExecutableName() {
+    static getJfExecutableName() {
+        return Utils.isWindows() ? 'jf.exe' : 'jf';
+    }
+    static getJFrogExecutableName() {
         return Utils.isWindows() ? 'jfrog.exe' : 'jfrog';
     }
     static isWindows() {
         return os.platform().startsWith('win');
     }
-    static runCli(cliPath, args) {
+    static runCli(args) {
         return __awaiter(this, void 0, void 0, function* () {
-            let res = yield exec_1.exec(cliPath, args);
+            let res = yield exec_1.exec('jf', args);
```

Comment:
What about versions prior to the jf executable?

Reply:
I added a comment explaining this.
It should work because this Action always downloads the 'jfrog' executable and cache it as 'jf' **and** 'jfrog'.